src/ray/gcs/tables.cc

```diff
@@ -415,6 +421,7 @@ template class Table<TaskID, ray::protocol::Task>;
 template class Table<TaskID, TaskTableData>;
 template class Log<ActorID, ActorTableData>;
 template class Log<TaskID, TaskReconstructionData>;
+template class Table<ClientID, TaskLeaseData>;
```

Comment:
Does it make more sense for this to be a mapping from `ClientID` or from `TaskID`? Pulling down the whole log for a client could be really expensive, right? Also, we can't really garbage collect keys from tables that are indexed by `ClientID`, right?

Reply:
Ah yes, that's right.